packages/cdktf-cli/lib/cdktf-project.ts

```diff
@@ -523,7 +529,7 @@ export class CdktfProject {
         }
 
         method === "deploy"
-          ? nextRunningExecutor.deploy()
+          ? nextRunningExecutor.deploy(refreshOnly)
```

Comment:
I think we should throw an Error if refreshOnly is set but method is destroy. While this does not happen when users use the cdktf-cli, it could happen as soon as someone uses the CdktfProject class directly. I think being safe here would be worth it :)